vizro-core/examples/kpi/app.py

```diff
@@ -0,0 +1,271 @@
+"""Example to show dashboard configuration."""
+
+import pandas as pd
+import vizro.models as vm
+from utils._charts import COLUMN_DEFS, KPI, bar, chloropleth, line, pie
+from utils._helper import clean_data_and_add_columns
+from vizro import Vizro
+from vizro.actions import filter_interaction
+from vizro.tables import dash_ag_grid
+
+# DATA --------------------------------------------------------------------------------------------
+df_complaints = pd.read_csv("data/Financial Consumer Complaints.csv")
```

Comment:
Could we not download this data maybe directly? Could be cool if there was no data connection needed

Reply:
https://github.com/mckinsey/vizro/pull/505#discussion_r1624975132

Reply:
Oh I meant actually directly fetching the data, not uploading the data.

```python
import pandas as pd
df = pd.read_csv('https://query.data.world/s/glbdstahsuw3hjgunz3zssggk7dsfu?dws=00000')
```

I checked the above and it works :)

Reply:
Ah nice, perfect! Thank you! ⭐ I thought that didn't work, because I had to create an account on data.world when I wanted to download the data manually, but apparently that's not required 😆